apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/db/Event.java

```diff
@@ -0,0 +1,36 @@
+package edu.pw.apsienrollment.event.db;
+
+import edu.pw.apsienrollment.user.db.User;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity(name = "Event")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Event {
```

Comment:
Please, add start of first meeting of the event and end of last meeting, as fields. You can fetch them dynamically using @Formula annotation as described here:
https://stackoverflow.com/questions/4912923/java-hibernate-mapping-property-with-a-query

Reply:
My mistake, done.